orttraining/orttraining/models/bert/main.cc

```diff
@@ -565,7 +566,8 @@ void setup_training_params(BertParameters& params) {
   size_t cuda_mem_limit = std::numeric_limits<size_t>::max();
   if (params.cuda_mem_limit_in_gb > 0)
     cuda_mem_limit = static_cast<size_t>(params.cuda_mem_limit_in_gb * 1024 * 1024 * 1024);
```

Comment:
I know it's not part of your change, but wouldn't this be very easy to overflow? Is there any checking on the input value?